app/presenters/organization_presenter.rb

```diff
@@ -0,0 +1,9 @@
+class OrganizationPresenter < Burgundy::Item
+  def members
+    UserPresenter.wrap item.members.sort_by(&:last_name)
+  end
+
+  def developments
```

Comment:
Because an organization's developments should always return unique records, this is less presentation logic and more model logic, so move this into the model. I recall you saying that [scoping the relation with distinct](http://guides.rubyonrails.org/association_basics.html#distinct) is out of the question, but I think I've seen `-> { uniq }` in a scope before, or you can overwrite the `developments` method on the model.